src/pages/home/wallet/SongRoyaltiesList.tsx

```diff
@@ -45,11 +52,12 @@ export default function SongRoyaltiesList({
   page,
   setPage,
 }: SongRoyaltiesListProps) {
-  const headerHeight = 430;
+  const windowHeight = useWindowDimensions()?.height;
+  const tableRef = useRef<any>();
```

Comment:
It might be better to add the eslint no-explicit-any to just this one line and not the whole file